railties/lib/rails/generators/rails/app/templates/config/cable.yml

```diff
@@ -6,5 +6,5 @@ test:
 
 production:
   adapter: redis
-  url: redis://localhost:6379/1
+  url: <%%= ENV.fetch("REDIS_URL") { "redis://localhost:6379/1" } %>
```

Comment:
Oh, right, we're inside a template which is ERB-processed too. Sorry, everything is correct)

That's the way it works–[link](https://docs.puppet.com/puppet/5.0/lang_template_erb.html#literal-tag-delimiters).